.github/workflows/formik-material-ui-pickers.yaml

```diff
@@ -1,33 +0,0 @@
-name: Build formik-material-ui
```

Comment:
we should leave this intact in case someone wants to patch the library for users still on 4.

We should update the peer deps to be less than or equal to material-ui@4

Reply:
On second this will play hell with the yarn workspaces. Guess we kill it